config/routes.rb

```diff
@@ -29,6 +29,8 @@
 
   resources :pattern_library, only: %w[index]
 
+  get 'reports/:id' => redirect { |params| "#{ENV['SALESFORCE_INSTANCE_URL']}/#{params[:id]}?csv=1" }
```

Comment:
@chrisdolendo @callaghanc I am trying to do the redirect here but I don't know if we have access to whether or not a user is an admin here and this needs to be ENV['COMMUNITY_LOGIN_URL'] if they're not an admin